Cubical/Data/HomotopyGroup/Base.agda

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.HomotopyGroup.Base where
+
+open import Cubical.Core.Prelude
+open import Cubical.Data.Nat
+open import Cubical.HITs.SetTruncation
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Group.Base
+import Cubical.Foundations.GroupoidLaws as GL
+
+Pointed : ∀ {ℓ} → Set (ℓ-suc ℓ)
```

Comment:
We might want to make a separate file for pointed types in Foundations?

Reply:
@mortberg Didn't #105 add some pointed stuff too?

Reply:
Yes, some of that should be upstreamed to the proper file.